test/cli/package-autocorrect-missing-import/test.out

```diff
@@ -23,6 +23,10 @@ foo_class.rb:15: Unable to resolve constant `MyClass` https://srb.help/5002
       NN |class Digest::Class
           ^^^^^^^^^^^^^^^^^^^
 
+foo_class.rb:17: `Test::Foo::Bar::OtherPackage::TestUtil` is defined in a test namespace and cannot be referenced in a non-test file https://srb.help/3720
+    17 |  Test::Foo::Bar::OtherPackage::TestUtil
+          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
Why do we get a new error here?

Reply:
>I suppose that's expected, because we're deferring the "missing import" errors until later in the pipeline now.

Exactly! The changes to the `.out` files do not generate any new error, and do not remove any existing errors. The difference is in the order of errors (because we changed the reporting order, as you've noticed), and some errors don't have autocorrects anymore, because otherwise it would lead to double importing.